### PR TITLE
[tencentcloudexporter]Fix some configuration file issues

### PR DESCRIPTION
**Pull request by wgliang**
**Description:** <Describe what has changed.>
Fix some configuration file issues.

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: exporter/tencentcloudlogserviceexporter/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This exporter supports sending OpenTelemetry log data to [LogService](https://cl
 
 # Configuration options:
 
-- `endpoint` (required): LogService's [Endpoint](https://cloud.tencent.com/document/product/614/56473).
+- `region` (required): LogService's [Region](https://cloud.tencent.com/document/product/614/56473).
```
Comment (codeboten) [inline exporter/tencentcloudlogserviceexporter/README.md:7]: This should be recorded in the changelog as a breaking change correct?

File: exporter/tencentcloudlogserviceexporter/README.md [hunk 2]
```diff
@@ -15,19 +15,33 @@ This exporter supports sending OpenTelemetry log data to [LogService](https://cl
...
-    traces:
-      receivers: [examplereceiver]
+    logs:
+      receivers: [otlp]
       exporters: [tencentcloud_logservice]
-```
\ No newline at end of file
+```
+
+# Changelog
```
Comment (jpkrohling) [inline exporter/tencentcloudlogserviceexporter/README.md:44]: I believe @codeboten meant the main changelog from this repository:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/87ba2fa130a3b1353d717499e3b62c7bea4924a0/CHANGELOG.md?plain=1#L3-L7

Comment (wgliang) [reply exporter/tencentcloudlogserviceexporter/README.md:44]: Get.

Comment (jpkrohling) [reply exporter/tencentcloudlogserviceexporter/README.md:44]: I would remove this changelog from the local readme, but it's your call :-) I'll merge it as-is once the CI allows, feel free to send another PR to remove it if you want.

**Review by jpkrohling — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**